### DISCUSSION TITLE
Passing long body to issues/new?title=body=

### DISCUSSION THREAD

**Discussion by wei** [community/community#22946] [New to GitHub]
<p>RE: About automation for issues and pull requests with query parameters</p>
<p>Hi I am looking at utilizing info from <a>this page</a> to build a custom issue creation feature for my users.</p>
<p>Basically when an error occurs, users can click a “Report an error” button which will redirect users to the github issue creation page with “body” prefilled with tracebacks and debug logs, etc.</p>
<p>Depending on the nature of error, body can be quite lengthy when raw debug outputs are attached which currently results in a 414 Request-URI Too Large error.</p>
<p>Is there anyway for me to pass a long body into the the issues endpoint (maybe a POST request?), I imagine long body in Github Issues is something that can be anticipated.</p>
<p>Really appreciate any help and ideas : P !</p>

**Comment by lee-dohm** [comment]
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/wei">@wei</a>,</p>
<p>Thanks for being part of the Community Forum!</p>
<p>There’s no direct workaround for using URL parameters that are too long. However, if you can provide more details about your specific workflow and goals, you might be able to use an alternative.</p>
<p>Specifically, if you need to include specific browser info, it may be easier for you to create a POST request to the API to create an issue and offer the user directions on adding additional information in a comment. Of course, if your workflow relies on author data for anything, that may not be the best solution.</p>
<p>On the other hand, if youre not grabbing in-the-moment data, you could use an <a href="https://help.github.com/articles/creating-issue-templates-for-your-repository/" rel="nofollow noopener">issue template</a> instead of the URL parameters.</p>
<p>One other option: If all you’re collecting is browser info, you could workaround the URL parameter issue by using a service like a <a href="http://whatsmybrowser.org" rel="nofollow noopener">whatsmybrowser.org</a> URL, instead of including all the specific info in the URL.</p>
<p>I hope this helps!</p>
